spec/models/topic_tracking_state_spec.rb

```diff
@@ -354,6 +354,31 @@
     expect(report.length).to eq(1)
   end
 
+  it "correctly handles indirectly muted categories" do
+    parent_category = Fabricate(:category)
+    sub_category = Fabricate(:category, parent_category_id: parent_category.id)
+    create_post(category: sub_category)
+
+    report = TopicTrackingState.report(user)
+    expect(report.length).to eq(1)
+
+    CategoryUser.create!(user_id: user.id,
+                         notification_level: CategoryUser.notification_levels[:muted],
+                         category_id: parent_category.id
+                         )
```

Comment:
```suggestion
    CategoryUser.create!(
      user_id: user.id,
      notification_level: CategoryUser.notification_levels[:muted],
      category_id: parent_category.id
    )
```

I think this format was from pre-rubocop days?